test/rules_prep/sometimes_true.py

```diff
@@ -4,8 +4,14 @@
 
 CAUSE_ID = 1003
 
-random.seed(0)
-
+# random.seed(0) # FIXME: this random value changed from python 2 to python 3, leading the test to fail
+# HACK: copy the random stream from python 2 into a list, to replicate the random stream from python 2 for testing purposes
```

Comment:
Lollllllllll nice (but scary). If it uses random behavior but requires consistent values, what was this actually testing?